README.md

```diff
@@ -0,0 +1,47 @@
+[![Build Status](https://travis-ci.org/weldr/haskell-rpm.svg?branch=master)](https://travis-ci.org/weldr/haskell-rpm)
+[![Coverage Status](https://coveralls.io/repos/github/weldr/haskell-rpm/badge.svg?branch=master)](https://coveralls.io/github/weldr/haskell-rpm?branch=master)
+
+Haskell library for working with RPM packages.
+
+
+Preparing local development environment for Haskell
+===================================================
+
+For development we use the latest upstream versions:
+
+1) Remove the standard `haskell-platform` and `ghc-*` RPMs if you have them installed
+2) Download version **8.0.2** of the generic Haskell Platform distribution from
+   https://www.haskell.org/platform/linux.html#linux-generic
+3) ```
+$ tar -xzvf haskell-platform-8.0.2-unknown-posix--minimal-x86_64.tar.gz 
+$ sudo ./install-haskell-platform.sh
+```
+4) Add `/usr/local/bin` to your PATH if not already there!
+
+
+Building the project locally
+============================
+
+`cabal` is used to install and manage Haskell dependencies from upstream.
+
+    $ cabal install
```

Comment:
Add `cabal sandbox init` so that it installs it in a per-project sandbox instead of the user's .cabal